src/redux/app-reducer.ts

```diff
@@ -1,12 +1,15 @@
 import { combineReducers, createStore } from 'redux'
 import { CounterReducer, ICounterReducerType } from './counter/counter.reducer'
+import { LateralAccessMenuReducer, ILateralAccessMenuReducerType } from './lateral-access-menu/lateral-access-menu.reducer'
 
 export interface IAppState {
-  counterReducer: ICounterReducerType
+  counterReducer: ICounterReducerType,
+  lateralAccessMenuReducer: ILateralAccessMenuReducerType,
```

Comment:
This last comma is called a trailing comma, it's used with the purpose to make a cleaner diff when adding a new element to the object since it doesn't require a comma in the previous line. The reason why it wasn't there is that I forgot to put it